client/jetpack-connect/style.scss

```diff
@@ -686,3 +686,21 @@
 		padding-right: 319px;
 	}
 }
+
+.jetpack-connect__plan-info {
+	text-align: center;
+	max-width: 500px;
+	margin-top: 20px;
+	padding-top: 30px;
```

Comment:
Could we combine the 2 padding declarations in one? `padding: 30px 24px` or maybe make it equal on all sides (`padding: 24px`) if @joanrho is good with that.